lib-dmtf/model/ComputerSystem_test.go

```diff
@@ -23,7 +23,7 @@ import (
+var CSystemJSON = "{\"@odata.context\":\"/redfish/v1/$metadata#ComputerSystem.ComputerSystem\",\"@odata.id\":\"/redfish/v1/Systems/1/\",\"@odata.type\":\"#ComputerSystem.v1_14_1.ComputerSystem\",\"@odata.etag\":\"W/3356AE4A\", \"Id\":\"1\",\"Description\":\"\",\"Name\":\"Computer System\",\"AssetTag\":\"\",\"BiosVersion\":\"U30 v1.46 (10/02/2018)\",\"HostName\":\"\",\"IndicatorLED\":\"Off\",\"Manufacturer\":\"HPE\",\"Model\":\"ProLiant DL380 Gen10\",\"PartNumber\":\"\",\"PowerRestorePolicy\":\"\",\"PowerState\":\"On\",\"SerialNumber\":\"2M291101JZ\",\"SKU\":\"868704-B21\",\"SubModel\":\"\",\"SystemType\":\"Physical\",\"UUID\":\"37383638-3430-4D32-3239-313130314A5A\",\"HostingRoles\":null,\"PCIeDevices\":null,\"PCIeFunctions\":null,\"Bios\":{\"@odata.id\":\"/redfish/v1/systems/1/bios/\"},\"Boot\":{\"AliasBootOrder\":null,\"BootNext\":\"\",\"BootOptions\":{\"@odata.id\":\"\"}, \"BootOrder\":null,\"BootOrderPropertySelection\":\"\",\"BootSourceOverrideEnabled\":\"Disabled\",\"BootSourceOverrideMode\":\"UEFI\",\"BootSourceOverrideTarget\":\"None\",\"Certificates\":{\"@odata.id\":\"\"},\"UefiTargetBootSourceOverride\":\"None\"},\"EthernetInterfaces\":{\"@odata.id\":\"/redfish/v1/Systems/1/EthernetInterfaces/\"},\"HostedServices\":{\"Oem\":{},\"StorageServices\":{\"@odata.id\":\"\"}},\"HostWatchdogTimer\":{\"FunctionEnabled\":false,\"Oem\":{},\"Status\":{\"Health\":\"\",\"State\":\"\",\"Oem\":{}},\"TimeoutAction\":\"\",\"WarningAction\":\"\"},\"Links\":{\"Chassis\":[{\"@odata.id\":\"/redfish/v1/Chassis/1/\"}],\"ManagedBy\":[{\"@odata.id\":\"/redfish/v1/Managers/1/\"}]},\"LogServices\":{\"@odata.id\":\"/redfish/v1/Systems/1/LogServices/\"},\"Memory\":{\"@odata.id\":\"/redfish/v1/Systems/1/Memory/\"},\"MemoryDomains\":{\"@odata.id\":\"\"},\"MemorySummary\":{\"MemoryMirroring\":\"\",\"TotalSystemMemoryGiB\":384,\"TotalSystemPersistentMemoryGiB\":0,\"Status\":{\"Health\":\"\",\"HealthRollup\":\"OK\",\"State\":\"\",\"Oem\":{}}},\"NetworkInterfaces\":{\"@odata.id\":\"/redfish/v1/Systems/1/NetworkInterfaces/\"},\"Processors\":{\"@odata.id\":\"/redfish/v1/Systems/1/Processors/\"},\"ProcessorSummary\":{\"Count\":2,\"LogicalProcessorCount\":0,\"Model\":\"Intel(R) Xeon(R) Gold 6152 CPU @ 2.10GHz\",\"Metrics\":{\"@odata.id\":\"\"},\"Status\":{\"Health\":\"\",\"HealthRollup\":\"OK\",\"State\":\"\",\"Oem\":{}}},\"Redundancy\":[{\"@odata.id\":\"\"}],\"SecureBoot\":{\"@odata.id\":\"/redfish/v1/Systems/1/SecureBoot/\"},\"SimpleStorage\":{\"@odata.id\":\"\"},\"Status\":{\"Health\":\"OK\",\"State\":\"Starting\",\"Oem\":{}},\"Storage\":{\"@odata.id\":\"/redfish/v1/Systems/1/Storage/\"},\"TrustedModules\":[{\"FirmwareVersion\":\"\",\"FirmwareVersion2\":\"\",\"InterfaceType\":\"\",     \"InterfaceTypeSelection\":\"\",\"Oem\":{},\"Status\":{\"Health\":\"\",\"State\":\"Absent\",\"Oem\":{}}}]}"
```

Comment:
Please replace version with string constant defined

Reply:
this file is not used by odimra